app/helpers/sufia_helper.rb

```diff
@@ -5,25 +5,21 @@ module SufiaHelper
 
   # override SufiaHelperBehavior method for institutional_access? condition
   def sufia_thumbnail_tag(document, options)
-    # collection
     if document.collection?
-      content_tag(:span, "", class: "glyphicon glyphicon-th collection-icon-search")
```

Comment:
It depends on who you ask. Rubocop tries to enforce single-quotes-unless-using-interpolation, and I guess I've internalized that because it makes us consistent with the rules the Hydra community would be following.

But it looks like [other people hate that](https://github.com/bbatsov/ruby-style-guide/issues/96) and [Github's styleguide disagrees entirely](https://github.com/styleguide/ruby/strings), so I guess the great thing about standards is there's so many to choose from.